sw/tools/airframe_editor/xml_airframe.py

```diff
@@ -76,13 +92,18 @@ def reorganize_airframe_xml(airframe_xml):
     find_and_add_sections_with_name(airframe_xml, airframe, "XSENS")
 
     find_or_add_group(airframe_xml, airframe, "GAINS")
+
     # Fixedwing
-    find_and_add_sections_with_name(airframe_xml, airframe, "HORIZONTAL CONTROL")
+    find_and_add_sections_with_name(airframe_xml, airframe,
```

Comment:
If that is okay with you, I will update the CONTRIBUTING.md file with elements from our discussion for Python and from info gleaned on the wiki (mostly for C). About OCaml (currently learning it), I found some standards (e.g. https://caml.inria.fr/resources/doc/guides/guidelines.en.html) but I have no idea which one is preferred on the project.